### PR TITLE
Support markdown in descriptions

### LINKED ISSUE #2: Render markdown as rich text
**Is your feature request related to a problem? Please describe.**
Per the OpenAPI spec, the description field supports rich text via CommonMark markdown syntax, and indeed many API definitions make use of this.

**Describe the solution you'd like**
I'd like the descriptions rendered as rich text.

**Describe alternatives you've considered**
It seems possible to use HTML inside the markdown content, but that defeats the purpose of using markdown :)

**Additional context**
Some screenshots:

Current output:
Screenshot 2021-01-06 at 22 37 43

Desired output:
Screenshot 2021-01-06 at 22 35 27

### PR DESCRIPTION
**Pull request by waldyrious**
Fixes #2, using Jekyll's native `markdownify` [filter](https://jekyllrb.com/docs/liquid/filters/). Some markdown formatting is introduced in the example OpenAPI definition, for demonstration.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@ <h1 class="offset-top" id="introduction">
       <small class="ml-2 text-muted">v{{- api.info.version -}}</small>
     </h1>
     {%- if api.info.description -%}
-      <p class="lead">{{- api.info.description -}}</p>
+      <p class="lead">{{- api.info.description | markdownify -}}</p>
```
Comment (robertlove) [inline index.html]: Change `<p>` to `<div>`

File: index.html [hunk 2]
```diff
@@ -108,7 +108,7 @@ <h1 class="h4 offset-top">{%- include utils/titleize.html word=collection -%}</h
             <h2 class="h5 mb-3 offset-top" id="{%- include utils/idify.html text=id -%}">{% include utils/request_method.html request_method=request_method %} {{ path[0] }}</h2>
 
             {%- if description -%}
-              <p>{{- description | newline_to_br -}}</p>
+              <p>{{- description | markdownify -}}</p>
```
Comment (robertlove) [inline index.html]: Change `<p>` to `<div>`

**Review by robertlove — CHANGES_REQUESTED** [review]
Hey @waldyrious, this looks good. Only changes I've requested are relating to changing `<p>` tags to `<div>` tags wherever Markdown is present. The reason for this is `mardownify` will generate `<p>` tags and this will result in nested `<p>` tags which results in invalid HTML.

**Comment by waldyrious** [comment]
@robertlove thanks for the quick review and astute observation. I've made the changes you suggested, and included additional details in the commit message for future reference.

File: index.html (hunk 1 above)
Comment (waldyrious) [reply index.html]: [Done](https://github.com/robertlove/jekyll-openapi/compare/ef3938c60162179b98d344d7d399305503da7d0f..39678c89071c2531c37b76b5730126c8a2d02236).

File: index.html (hunk 2 above)
Comment (waldyrious) [reply index.html]: [Done](https://github.com/robertlove/jekyll-openapi/compare/ef3938c60162179b98d344d7d399305503da7d0f..39678c89071c2531c37b76b5730126c8a2d02236).

**Pull request merged**